src/components/about-us/section-05/timeline.js

```diff
@@ -0,0 +1,176 @@
+import PropTypes from 'prop-types'
+import React, { PureComponent } from 'react'
+import styled, { keyframes } from 'styled-components'
+
+const restart = keyframes`
+  0%, 100% {
+    opacity: 1;
+  }
+  50% {
+    opacity: 0;
+    transform: translate3d(0, 0, 0);
+  }
+`
+
+const Container = styled.div`
+  width: 100%;
+  height: 100%;
+  animation: ${props => props.returnToStart ? `${restart} 1s linear` : 'none'};
+  cursor: grab;
+  cursor: -moz-grab;
+  cursor: -webkit-grab;
+  user-select: none;
+`
+
+const Scroller = styled.div.attrs({
+  style: ({ shiftY }) => ({
+    transform: `translate3d(0, -${shiftY}px, 0)`
+  })
+}) `
+  width: 100%;
+`
+
+export default class Timeline extends PureComponent {
+  constructor(props) {
+    super(props)
+    this.state = {
+      frameId: null,
+      timelineShiftY: 0,
+      returnToStart: false,
+      mouseDown: false,
+      prevClientY: null
+    }
+  }
+
+  _startLoop = () => {
+    const frameId = window.requestAnimationFrame(this._loop)
+    this.setState({ frameId: frameId })
+  }
+
+  _loop = () => {
+    this._setAutoScroll()
+    const frameId = window.requestAnimationFrame(this._loop)
+    this.setState({ frameId: frameId })    
```

Comment:
Do we need to put `frameId` in `this.state`? 
Or just put it in the `this` global object?